x-pack/plugins/apm/public/components/alerting/rule_types/error_count_rule_type/index.tsx

```diff
@@ -101,6 +105,9 @@ export function ErrorCountRuleType(props: Props) {
                 start,
                 end,
                 groupBy: params.groupBy,
+                useFilterQuery:
+                  params.useFilterQuery === true ? 'true' : 'false',
+                filterQuery: params.filterQuery,
```

Comment:
`useFilterQuery` is not needed here. And `useFilterQuery` can be renamed to `kqlFilter`

```suggestion
                kqlFilter: params.kqlFilter,
```